bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -409,7 +409,9 @@ spec:
           - delete
         serviceAccountName: windows-machine-config-operator
       deployments:
-      - name: windows-machine-config-operator
+      - label:
```

Comment:
can remove this change

Reply:
happy to drop, but any reason we don't want the label? it was generated with `make bundle`